File: C11/백준/Bronze/23235. The Fastest Sorting Algorithm In The World/The Fastest Sorting Algorithm In The World.c
#include <stdio.h>

#define MAX 1000

int main(void){
	
	int number = 1;
	char input[MAX];
	
	while(fgets(input, MAX, stdin)){
		if(input[0] == '0'){
			break;
		}
		printf("Case %d: Sorting... done!\n", number);
		number++;
	}
	
	return 0;
}

File: Java/백준/Bronze/23235. The Fastest Sorting Algorithm In The World/The Fastest Sorting Algorithm In The World.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int number = 1;
        String input;

        while (true) {
            input = br.readLine();
            if (input.equals("0")) {
                break;
            }
            System.out.println("Case " + number + ": Sorting... done!");
            number++;
        }
    }
}